addons/hr_holidays/models/hr.py

```diff
@@ -82,6 +83,70 @@ class Employee(models.Model):
     show_leaves = fields.Boolean('Able to see Remaining Leaves', compute='_compute_show_leaves')
     is_absent_totay = fields.Boolean('Absent Today', compute='_compute_absent_employee', search='_search_absent_employee')
 
+    leave_ids = fields.One2many('hr.leave', 'employee_id', readonly=True)
```

Comment:
CodingRemark: I would move those fields on top of fields declaration (first o2m to leaves and allocation then fields computed "based on those" (even if the computation bypasses them a bit)).